e2e/test/scenarios/embedding/embedding-downloads-questions.cy.spec.js

```diff
@@ -0,0 +1,142 @@
+import {
+  restore,
+  describeEE,
+  visitQuestion,
+  visitEmbeddedPage,
+  popover,
+  setTokenFeatures,
+  visitIframe,
+  filterWidget,
+} from "e2e/support/helpers";
+
+const questionDetails = {
+  name: "Simple SQL Query for Embedding",
+  native: {
+    query: "select {{text}} as WYSIWYG",
+    "template-tags": {
+      text: {
+        id: "fake-uuid",
+        name: "text",
+        "display-name": "Text",
+        type: "text",
+        default: null,
+      },
+    },
+  },
+};
+
+describeEE("scenarios > embedding > questions > downloads", () => {
+  beforeEach(() => {
+    cy.intercept("PUT", "/api/card/*").as("publishChanges");
+    cy.intercept("GET", "/api/embed/card/**/query").as("dl");
```

Comment:
Unused alias. Will it be used in future PRs?

```suggestion
    cy.intercept("GET", "/api/embed/card/**/query").as("download");
```

Reply:
@kamilmielnik sorry, I'm only now getting back to this PR.

Yes, this will be used in the next PR as the mechanism to get the actual download.